app/infrastructure/exceptions/__init__.py

```
@@ -1,5 +1,12 @@
 from .error_type import ErrorType
 from .scraper_exception import ScraperException
+from .db_exception import DatabaseException, DatabaseIntegrityException
 
-__all__ = ["ClientException", "ErrorType", "ScraperException"]
+__all__ = [
+    "ClientException",
+    "ErrorType",
+    "ScraperException",
+    "DatabaseException",
+    "DatabaseIntegrityException",
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)